src/UIComponents/GenerateResults.jsx

```diff
@@ -181,14 +181,23 @@ class GenerateResults extends Component {
           }}
         >
           <div className="ailab-image-hover" style={styles.trainBot}>
-            <img src={aiBotHead} style={styles.trainBotHead} />
-            <img src={aiBotBody} style={styles.trainBotBody} />
+            <img
+              src={aiBotHead}
+              style={styles.trainBotHead}
+              alt="A.I. bot head"
```

Comment:
Alt tags are definitely an art, not a science, so this is just a suggestion, but I wonder if there is a way for us to just summarize the three "A.I bot head/body/scanner beam" alt-texts to have just one alt-text that's like "A.I Bot with Scanner Beam" or something like that. From my point of view, that would summarize the idea of what is happening similar to how a sighted person would take in all three pieces as a whole.